workspaces/quay/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@internal/quay",
+  "version": "1.0.0",
+  "private": true,
+  "engines": {
+    "node": "18 || 20"
+  },
+  "scripts": {
+    "tsc": "tsc",
+    "tsc:full": "tsc --skipLibCheck true --incremental false",
+    "build:all": "backstage-cli repo build --all",
+    "build:api-reports": "yarn build:api-reports:only",
+    "build:api-reports:only": "backstage-repo-tools api-reports --allow-all-warnings -o ae-wrong-input-file-type, ae-undocumented --validate-release-tags",
+    "clean": "backstage-cli repo clean",
+    "test": "backstage-cli repo test",
+    "test:all": "backstage-cli repo test --coverage",
+    "fix": "backstage-cli repo fix",
+    "lint": "backstage-cli repo lint --since origin/main",
+    "lint:all": "backstage-cli repo lint",
+    "prettier:check": "prettier --check .",
+    "new": "backstage-cli new --scope @backstage-community",
+    "postinstall": "cd ../../ && yarn install"
+  },
+  "workspaces": {
+    "packages": [
+      "packages/*",
+      "plugins/*"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/community-plugins",
+    "directory": "workspaces/quay"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.28.0",
+    "@backstage/e2e-test-utils": "^0.1.1",
+    "@backstage/repo-tools": "^0.8.0",
+    "@changesets/cli": "^2.27.1",
+    "@ianvs/prettier-plugin-sort-imports": "^4.4.0",
+    "@spotify/prettier-config": "^12.0.0",
+    "node-gyp": "^9.0.0",
+    "prettier": "^2.3.2",
+    "react-router-dom": "^6.28.0",
+    "typescript": "~5.3.0"
+  },
+  "dependencies": {
+    "knip": "^5.27.4"
+  },
+  "resolutions": {
+    "@types/react": "^18",
+    "@types/react-dom": "^18",
+    "@microsoft/api-extractor": "7.36.4"
```

Comment:
Just one more thing, Can u try removing "@microsoft/api-extractor": "7.36.4" and applying this change: https://github.com/backstage/community-plugins/pull/1897/files/77d3acd77572007cf54a605fe26eff284498edfe#r1834470768